src/actions/guides/json_and_apis/saving-to-the-database.cr

```diff
@@ -1,4 +1,5 @@
 class Guides::JsonAndApis::SavingToTheDatabase < GuideAction
+  ANCHOR_SAVING_TO_THE_DATABASE = "saving-to-the-database_"
```

Comment:
And maybe rather than end with `_` it could be prefixed with `perma-`

Reply:
That's fine too. That's just what will show up in the URL. Not sure if that's a factor when it comes to SEO or not.

Reply:
I think that is ok to show in the URL and hopefully won't affect SEO :P